src/main/java/com/google/devtools/build/lib/bazel/bzlmod/YankedVersionsValue.java

```diff
@@ -39,13 +39,13 @@ public static YankedVersionsValue create(Optional<ImmutableMap<Version, String>>
   abstract static class Key implements SkyKey {
     private static final SkyKeyInterner<Key> interner = SkyKey.newInterner();
 
-    abstract String getModuleName();
+    abstract ModuleKey getModuleKey();
```

Comment:
this change is rather confusing. I vaguely understand why you did it but ultimately this is the "yanked version***s*** value", which only makes sense for a module name, not a specific module version.

it's getting late here and my brain is too scrambled to come up with a sensible alternative, but I'm flagging this anyway for discussion.